apps/keycloak/vars/all/main.yml

```diff
@@ -13,7 +13,8 @@ keycloak_database_secret_name: "keycloak-database-{{ keycloak_vault_checksum | d
 
 # -- keycloak
 keycloak_image_name: "quay.io/keycloak/keycloak"
-keycloak_image_tag: "latest"
+# FIXME
+keycloak_image_tag: "legacy"
```

Comment:
I looked at the repository, what the difference between the tag `17.0.0-legacy` and `17.0.0` ? Because `legacy` and `17.0.0-legacy` have the same hash. 
You commit message suggest to not use the version `17.0.0` of keycloak but you are using it in fact.

Reply:
I'm using the legacy version of the `17.0.0` release. This version does not use the new configuration pattern and is thus functional with the current tray. Is that more clear?

Reply:
Yes thanks